### ISSUE TITLE
[1.9.6] iPhone 6 / iOS 9.3.3 - Error 11800 - Create a lomotif with slo mo video of length 4 sec and drag the time slider to 30 sec

### ISSUE THREAD

**Issue by hiteshbheda** [Lomotif/lomotif-ios-issues#561]
Frequency: Every time

Action:
1. Shoot a slo mo video of length 4 sec
2. Launch Lomotif
3. Click on Start
4. Select slo mo video (4 sec length) from camera roll
5. Don't add any song
6. At create screen tap on Adjust tab
7. Drag the time slider to 30 sec
8. Check whether you are getting any error popup message on the screen?

Expected result: Lomotif should be created with 30 sec length

Actual result: Displays Popup Error 11800

![img_3677](https://cloud.githubusercontent.com/assets/9691390/17389250/1563ddd8-5a20-11e6-9af7-6c910fbbc202.PNG)

**Comment by xiuie** [comment]
Duplicate of #560 As spoken, wouldn't log this as all have same root cause, but please go through these tests again when new build is released.

**Comment by hiteshbheda** [comment]
@xiuie Agree. Sure, I'll test all these tests once the new build will be released.

**Comment by testershub** [comment]
@xiuie  Issue is reproducible on the latest build [1.9.7]

**Comment by vysohanych** [comment]
@xiuie I can reproduce it on iPhone 5S iOS 8.4.1 on build 1.9.7

**Comment by hiteshbheda** [comment]
@xiuie I can reproduce this issue for build [1.9.9] on iPhone 6 / iOS 9.3.4
